Extension/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 0.17.7: July 16, 2018
+* Add `${vcpkgRoot}` variable. [#1817](https://github.com/Microsoft/vscode-cpptools/issues/1817)
+* Skip automatic parsing of source files in Mac system framework paths. [#2156](https://github.com/Microsoft/vscode-cpptools/issues/2156)
+* Fix indexing of the entire root drive on Windows when no is folder open. [#2216](https://github.com/Microsoft/vscode-cpptools/issues/2216)
+* Fix infinite CPU/memory usage until it crashes with `#include` code actions when no folder is open. [#2225](https://github.com/Microsoft/vscode-cpptools/issues/2225)
```

Comment:
TMI - can you just say "infinite loop"?

Reply:
You mean "* Fix infinite loop." or something else?

Reply:
yes.
"Fix infinite loop with \`#include\` code actions when no folder is open."